my-portfolio/src/p5_sketches/ball.js

```diff
@@ -0,0 +1,19 @@
+function setup() {
+    var canvas = createCanvas(480, 320);
+    canvas.parent('1');
+    x = width/2;
+    y = height/2;
+    xSpeed = 1;
+    ySpeed = 1;
+}
+var x, y, xSpeed, ySpeed;
+
+function draw(){
+    background(0);
+    ellipse(x, y, 16,16);
+    x+=xSpeed;
+    y+=ySpeed;
```

Comment:
Is this your js file?

Reply:
yes, this is supposed to display the canvas and do the animation, but isn't working.